src/powershell-native/nativemsh/pwrshcommon/pwrshcommon.cpp

```diff
@@ -1408,8 +1407,7 @@ namespace NativeMsh
         // Fall back to attempt to load the CLR from the alternate inbox location
         // or if the ALC was not located in the CoreCLR directory.
```

Comment:
This comment should be updated as well

Reply:
Sure thing. Also removed `std::string assemblyListToSearch = assemblyList.str();` in the next line as it's not used anywhere.